codegen/arrai/service.arrai

```diff
@@ -5,11 +5,12 @@ let sysl = //{./sysl};
 
 # codegen groups
 
-let restClient = {"error_types", "service", "types", "types_extn"};
-let restService = restClient | {"requestrouter", "servicehandler", "serviceinterface", "types_extn"};
+let restClient = {"error_types", "service", "types"};
+let restService = restClient | {"requestrouter", "servicehandler", "serviceinterface"};
 
 let grpcClient = {"grpc_client"};
 let grpcService = grpcClient | {"grpc_handler", "grpc_interface"};
+let migrate = {"types"};
```

Comment:
I don't think `migrate` belongs here. `service.arrai` is designed to provide a simple mechanism for users to generate a set of files for given use cases. You could potentially derive a similar file for use within the test suite itself.

Reply:
I could principally have another copy of service.arrai within the test suite itself. But it would be similar to the existing service.arrai. It is better to have a single source of truth, otherwise there is a chance that the two copies of service.arrai would drift away from each other. We will endup merging the changes from one file into another.